main.go

```diff
@@ -58,6 +62,22 @@ func (ctx *corazaPlugin) OnPluginStart(pluginConfigurationSize int) types.OnPlug
 		return types.OnPluginStartStatusFailed
 	}
 
+	root, _ := fs.Sub(crs, "rules")
+
+	root = &rulesFS{
+		root,
+		map[string]string{
+			"@recommended-conf":    "coraza.conf-recommended.conf",
+			"@demo-conf":           "coraza-demo.conf",
```

Comment:
This PR was surprisingly hard to revert :) One issue is having trouble wrapping my head around now having `demo-`, `crs-setup-demo`, and `crs-setup-`. Can we merge `crs-setup-demo` into `demo-`? Having two demos seems like a bit much